packages/nextjs/components/Header.tsx

```diff
@@ -4,14 +4,28 @@ import Link from "next/link";
 import Image from "next/image";
 import { Faucet } from "~~/components/scaffold-eth";
 import RainbowKitCustomConnectButton from "~~/components/scaffold-eth/RainbowKitCustomConnectButton";
-import { BugAntIcon, SparklesIcon } from "@heroicons/react/24/outline";
+import { BugAntIcon, SparklesIcon, HomeModernIcon, BeakerIcon } from "@heroicons/react/24/outline";
+
+function checkAndCloseDropDown(e: { currentTarget: any }) {
+  const targetEl = e.currentTarget;
+  if (targetEl && targetEl.matches(":focus")) {
+    setTimeout(function () {
+      targetEl.blur();
+    }, 0);
+  }
+}
```

Comment:
This is not the way to this in React. For opening / closing the dropdown we can have a state.